blocks/library/list/index.js

```diff
@@ -72,6 +112,17 @@ registerBlock( 'core/list', {
 		return (
 			<Editable
 				tagName={ nodeName.toLowerCase() }
+				onConfig={ ( settings ) => ( {
```

Comment:
I changed onConfig() to getSettings() . 

I don't understand what you are trying to explain with the code snippet.